src/pages/[platform]/build-a-backend/graphqlapi/connect-api-to-existing-database/index.mdx

```diff
@@ -0,0 +1,670 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect your API to an existing database',
+  description:
+    'Learn how to connect your API to an existing MySQL or PostgreSQL database.',
+  platforms: ['javascript', 'react-native']
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps(context) {
+  return {
+    props: {
+      platform: context.params.platform,
+      meta
+    }
+  };
+}
+
+<InlineFilter filters={["javascript", "react-native"]}>
```

Comment:
make sure all platforms see this, not just js ones

Reply:
updated to include all platforms